src/openforms/emails/digest.py

```diff
@@ -179,3 +257,80 @@ def collect_broken_configurations() -> list[BrokenConfiguration]:
         )
 
     return broken_configurations
+
+
+def collect_invalid_registration_backends() -> list[InvalidRegistrationBackend]:
+    registration_backends = (
+        backend
+        for backend in FormRegistrationBackend.objects.select_related("form").iterator()
+        if backend.form.is_available
+    )
+
+    already_detected = []
+    invalid_registration_backends = []
+    for registration_backend in registration_backends:
+        form_name = registration_backend.form.admin_name
+        form = registration_backend.form
+        backend_options = registration_backend.options
+        backend_type = registration_backend.backend
+        plugin = register[backend_type]
+
+        # errors in general configuration are more straightforward for the user,
+        # so we show the exact ones
+        if plugin.verbose_name not in already_detected and (
+            broken_general_config := check_general_config(plugin)
+        ):
+            already_detected.append(plugin.verbose_name)
+            invalid_registration_backends.append(
+                InvalidRegistrationBackend(
+                    config_name=plugin.verbose_name,
+                    exception_message=broken_general_config,
+                    form_name=form_name,
+                )
+            )
+
+        # errors in the form level can be more detailed so here we want to show
+        # a more general error and the link to the broken form to investigate
+        if not is_form_config_valid(plugin, backend_options):
+            invalid_registration_backends.append(
+                InvalidRegistrationBackend(
+                    config_name=plugin.verbose_name,
+                    exception_message=_(
+                        "Invalid registration backend configuration has been detected at the form level"
+                    ),
+                    form_name=form_name,
+                    form_id=str(form.id),
+                    form_level=True,
+                )
+            )
+
+    return invalid_registration_backends
+
+
+def collect_invalid_logic_rules() -> list[InvalidLogicRule]:
+    forms = Form.objects.live()
+    static_variables = [var.key for var in get_static_variables()]
+
+    invalid_logic_rules = []
+    for form in forms:
+        form_variables_keys = [
+            form_variable.key for form_variable in form.formvariable_set.all()
+        ]
+
+        all_keys = static_variables + form_variables_keys
+        form_logics = FormLogic.objects.filter(form=form)
+        form_logics_vars = [
+            var for logic in form_logics for var in get_variables_from_logic(logic)
+        ]
+
+        for var in form_logics_vars:
```

Comment:
As we discussed, it might be that a variable is of type `object`, and a `"var"` json logic operator is being used to access a (nested) property of this object, e.g.:

`someStaticVar` is of type `object`, and we have a json rule with the json logic trigger:

```json
{"var": "someStaticVar.someProperty"}
```

In this case, this will add an `InvalidLogicRule` entry as a false positive. However, this case might be really uncommon.

Reply:
1. this should handle the array (for default value) too indeed
2. let's add some logging for when a variable key has dots in it and it's not found so that we can monitor for false positives. we even expose a `auth` static variable which is of type object that can cause problems here. Unfortunately, we're not far enough yet to provide more detailed schemas of the variables that we could make use of for valid/invalid variable specifications.

I think it'd be wise to not report on missing variables if a "parent" variable (`someStaticVar`) of type object exists.